apps/src/aiTutor/views/Message.tsx

```diff
@@ -76,6 +79,9 @@ const ChatMessage: React.FunctionComponent<ChatMessageProps> = ({message}) => {
 
       {isAssistant(message.role) && (
         <div className={style.assistantMessageContainer}>
+          <Typography semanticTag="h5" visualAppearance="heading-xs">
+            AI Tutor ({message.role})
```

Comment:
This header distinguishes the AI Tutor chats without a big circular image taking up horizontal bandwidth like we had before. 

<img width="771" alt="Screenshot 2024-04-10 at 11 24 35 PM" src="https://github.com/code-dot-org/code-dot-org/assets/26844240/7cbfdf78-2461-4cd8-8840-bdcd689be56d">